<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Publications</title>

<script src="site_libs/header-attrs-2.24/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>










<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Saur Lab</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Team
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="./current.html">Current team</a>
    </li>
    <li>
      <a href="./previous.html">Previous members</a>
    </li>
  </ul>
</li>
<li>
  <a href="./research.html">Research</a>
</li>
<li>
  <a href="./publications.html">Publications</a>
</li>
<li>
  <a href="./join.html">Join us</a>
</li>
<li>
  <a href="./contact.html">Contact</a>
</li>
<li>
  <a href="./news.html">News</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="./legal.html">Legal notice</a>
</li>
<li>
  <a href="https://twitter.com/izzysaur">
    <span class="fab fa-twitter"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore"><strong>Publications</strong></h1>

</div>


<p><br> <br> “corresponding <br> <br></p>
<div id="for-stats-you-may-want-to-look-at-google-scholar"
class="section level3">
<h3>For stats, you may want to look at <a
href="https://scholar.google.de/citations?user=u21upHAAAAAJ&amp;hl=en">Google
Scholar</a></h3>
<p><br> <br></p>
</div>
<div id="section" class="section level3">
<h3>2023</h3>
<p><br></p>
<p><strong>Crean EE, M Bilstein-Schloemer M</strong>, T Maekawa T, P
Schulze-Lefert P, <strong>Saur IML”</strong>. A dominant-negative
avirulence effector of the barley powdery mildew fungus provides
mechanistic insight to barley MLA immune receptor activation. <strong>J
Exp Botany</strong> doi: 10.1093/jxb/erad285</p>
<p><br></p>
<p><strong>Shi W</strong>, Stolze SC, Nakagami H, Misas Villamil JC,
<strong>Saur IML</strong>, Doehlemann G”. Combination of in-vivo
proximity labeling and co-immunoprecipitation identifies the host target
network of a tumor-inducing effector in the fungal maize pathogen
Ustilago maydis. <strong>J Exp Botany</strong> doi:
10.1093/jxb/erad188.</p>
<p><br></p>
<p>Yu G, Matny O, Gourdoupis S, Rayapuram N, Aljedaani FR, Wang YL,
Nürnberger T, Johnson R, <strong>Crean EE, Saur IML</strong>, Gardener
C, Yue Y, Kangara N, Steuernagel B, Hayta S, Smedley M, Harwood W,
Patpour M, Wu S, Poland J, Jones JDG, Reuber TL, Ronen M, Sharon A,
Rouse MN, Xu S, Holušová K, Bartoš J, Molnár I, Karafiátová M, Hirt H,
Blilou I, Jaremko Ł, Doležel J, Steffenson BJ”, Wulff BBH”. The wheat
stem rust resistance gene Sr43 encodes an unusual protein kinase.
<strong>Nature Genetics</strong> doi: 10.1038/s41588-023-01402-1</p>
<p><br></p>
<p>Wanke A, van Boerdonk S, Mahdi LK, Stephan Wawra S, Neidert M,
Chandrasekar B, Saake P, <strong>Saur IML</strong>, Derbyshire P, Holton
N, Menke FLH, Brands M, Pauly M, Acosta IF, Zipfel C, Zuccaro A”. A
GH81-type β-glucan-binding protein facilitates colonization by
mutualistic fungi in barley. <strong>bioRxiv</strong> <a
href="doi:10.1101/2023.04.12.536646"
class="uri">doi:10.1101/2023.04.12.536646</a></p>
<p><br> <br></p>
</div>
<div id="section-1" class="section level3">
<h3>2022</h3>
<p><br></p>
<p>Märkle H, <strong>Saur IML</strong>, Stam R”. Evolution of resistance
(R) gene specificity. <strong>Essays in Biochemistry</strong> <a
href="doi:10.1042/EBC20210077"
class="uri">doi:10.1042/EBC20210077</a></p>
<p><br></p>
<p>Ortiz D, Chen J, Outram MA, <strong>Saur IML</strong>, Upadhyaya NM,
Mago R, Ericsson DJ, Cesari S, Chen C, Williams SJ”, Dodds PN”. The stem
rust effector protein AvrSr50 escapes Sr50 recognition by a substitution
in a single surface-exposed residue. <strong>New Phytologist</strong> <a
href="doi:10.1111/nph.18011" class="uri">doi:10.1111/nph.18011</a></p>
<p><br></p>
<p><strong>Saur IML”</strong>, Lawson AW, Schulze-Lefert P”. Buy one,
get two. <strong>Nature Plants</strong> doi:
10.1038/s41477-022-01097-y</p>
<p><br> <br></p>
</div>
<div id="section-2" class="section level3">
<h3>2021</h3>
<p><br></p>
<p><strong>Saur IML”</strong>, Hückelhoven R”. <strong>Humboldt-Review
2021</strong>: Recognition and defence of plant-infecting fungal
pathogens. <strong>J Plant Physiology</strong> <a
href="doi:10.1016/j.jplph.2020.153324"
class="uri">doi:10.1016/j.jplph.2020.153324</a>.</p>
<p><br></p>
<p><strong>Saur IML</strong>, Panstruga R, Schulze-Lefert P”. NOD-like
receptor-mediated plant immunity: from structure to cell death.
<strong>Nature Reviews Immunology</strong> doi:
10.1038/s41577-020-00473-z</p>
<p><br></p>
<p>Bauer S, Yu D, Lawson AW, <strong>Saur IML</strong>, Frantzeskakis L,
Kracher B, Logemann E, Chai J, Maekawa T, Schulze-Lefert P”. The
leucine-rich repeats in allelic barley MLA immune receptors define
specificity towards sequence-unrelated powdery mildew avirulence
effectors with a predicted common RNase-like fold. <strong>PLoS
Pathogens</strong> doi: 10.1371/journal.ppat.1009223</p>
<p><br> <br></p>
</div>
<div id="section-3" class="section level3">
<h3>2020</h3>
<p><br></p>
<p>Mahdi LK, Huang M, Zhang X, Nakano RT, Kopp LB, <strong>Saur
IML</strong>, Jacob F, Kovacova V, Lapin D, Parker JE, Murphy JM,
Hofmann K, Schulze-Lefert P, Chai J, Maekawa T”. Discovery of a Family
of Mixed Lineage Kinase Domain-like Proteins in Plants and Their Role in
Innate Immune Signaling. <strong>Cell, Host &amp; Microbe</strong> doi:
10.1016/j.chom.2020.08.012.</p>
<p><br> <br></p>
</div>
<div id="section-4" class="section level3">
<h3>2019</h3>
<p><br></p>
<p><strong>Saur IML”</strong>, Bauer S, Lu X, Schulze-Lefert P”. A cell
death assay in barley and wheat protoplasts for identification and
validation of matching pathogen AVR effector and plant NLR immune
receptors. <strong>Plant Methods</strong> doi:
10.1186/s13007-019-0502-0.</p>
<p><br></p>
<p><strong>Saur IML</strong>, Bauer S, Kracher B, Lu X, Franzeskakis L,
Müller MC, Sabelleck B, Kümmel F, Panstruga R, Maekawa T, Schulze-Lefert
P”. Multiple pairs of allelic MLA immune receptor-powdery mildew AVRA
effectors argue for a direct recognition mechanism.
<strong>Elife</strong> doi: 10.7554/eLife.44471</p>
<p><br></p>
<p>Maekawa T, Kracher B, <strong>Saur IML</strong>, Yoshikawa-Maekawa M,
Kellner R, Pankin A, von Korff M, Schulze-Lefert P”. Subfamily-Specific
Specialization of RGH1/MLA Immune Receptors in Wild Barley. <strong>Mol
Plant Microbe Interact</strong> doi: 10.1094/MPMI-07-18-0186-FI.</p>
<p><br> <br></p>
</div>
<div id="section-5" class="section level3">
<h3>2018</h3>
<p><br></p>
<p>Conlan B”, Stoll T, Gorman JJ, <strong>Saur IML</strong>, Rathjen
JP”. Development of a Rapid in planta BioID System as a Probe for Plasma
Membrane-Associated Immunity Proteins. <strong>Front Plant Sci</strong>
doi: 10.3389/fpls.2018.01882</p>
<p><br> <br></p>
</div>
<div id="section-6" class="section level3">
<h3>2017</h3>
<p><br></p>
<p>Thynne E, <strong>Saur IML</strong>, Simbaqueba J, Ogilvie HA,
Gonzalez-Cendales Y, Mead O, Taranto A, Catanzariti AM, McDonald MC,
Schwessinger B, Jones DA”, Rathjen JP”, Solomon PS”. Fungal
phytopathogens encode functional homologues of plant rapid
alkalinization factor (RALF) peptides.<strong>Mol Plant Pathol</strong>
doi: 10.1111/mpp.12444.</p>
<p><br> <br></p>
</div>
<div id="section-7" class="section level3">
<h3>2016</h3>
<p><br></p>
<p>Lu X, Kracher B, <strong>Saur IML</strong>, Bauer S, Ellwood SR, Wise
R, Yaeno T, Maekawa T, Schulze-Lefert P”. Allelic barley MLA immune
receptors recognize sequence-unrelated avirulence effectors of the
powdery mildew pathogen. <strong>PNAS USA</strong> doi:
10.1073/pnas.</p>
<p><br></p>
<p>Pfeilmeier S, <strong>Saur IML</strong>, Rathjen JP, Zipfel C”,
Malone JG”. High levels of cyclic-di-GMP in plant-associated Pseudomonas
correlate with evasion of plant immunity. <strong>Mol Plant
Pathol</strong> doi: 10.1111/mpp.12297</p>
<p><br></p>
<p><strong>Saur IML</strong>, Kadota Y, Sklenar J, Holton NJ, Smakowska
E, Belkhadir Y, Zipfel C”, Rathjen JP”. NbCSPR underlies age-dependent
immune responses to bacterial cold shock protein in Nicotiana
benthamiana. <strong>PNAS USA</strong> doi: 10.1073/pnas.1511847113</p>
<p><br> <br></p>
</div>
<div id="section-8" class="section level3">
<h3>2015</h3>
<p><br></p>
<p><strong>Saur IML</strong>, Conlan BF, Rathjen JP”. The N-terminal
domain of the tomato immune protein Prf contains multiple homotypic and
Pto kinase interaction sites. <strong>J Biol Chem</strong> <a
href="doi:10.1074/jbc.M114.616532"
class="uri">doi:10.1074/jbc.M114.616532</a>.</p>
<p><br></p>
<p>Ntoukakis V, <strong>Saur IML</strong>, Conlan B, Rathjen JP”. The
changing of the guard: the Pto/Prf receptor complex of tomato and
pathogen recognition. <strong>Curr Opin Plant Biol</strong> doi:
10.1016/j.pbi.2014.04.002</p>
<p><br> <br></p>
</div>
<div id="section-9" class="section level3">
<h3>2011</h3>
<p><br></p>
<p><strong>Saur IML</strong>, Oakes M, Djordjevic MA, Imin N”. Crosstalk
between the nodulation signaling pathway and the autoregulation of
nodulation in Medicago truncatula. <strong>New Phytologist</strong> doi:
10.1111/j.1469-8137.2011.03738.</p>
<p><br> <br> <br></p>
<p>© Isabel Saur</p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
